cli/src/pcluster/cli.py

```diff
@@ -331,68 +336,21 @@ def _get_parser():
 
     # createami command subparser
     pami = subparsers.add_parser(
-        "createami", help="(Linux/macOS) Creates a custom AMI to use with AWS ParallelCluster."
-    )
-    pami.add_argument(
-        "-ai",
-        "--ami-id",
-        dest="base_ami_id",
-        required=True,
-        help="Specifies the base AMI to use for building the AWS ParallelCluster AMI.",
-    )
-    pami.add_argument(
-        "-os",
-        "--os",
-        dest="base_ami_os",
-        required=True,
-        help="Specifies the OS of the base AMI. "
-        "Valid options are: alinux, ubuntu1604, ubuntu1804, centos7, centos8.",
+        "build-image", help="(Linux/macOS) Creates a custom AMI to use with AWS ParallelCluster."
     )
     pami.add_argument(
         "-i",
-        "--instance-type",
-        dest="instance_type",
-        default="t2.xlarge",
-        help="Sets instance type to build the ami on. Defaults to t2.xlarge.",
-    )
-    pami.add_argument(
-        "-ap",
-        "--ami-name-prefix",
-        dest="custom_ami_name_prefix",
-        default="custom-ami-",
-        help="Specifies the prefix name of the resulting AWS ParallelCluster AMI.",
-    )
-    pami.add_argument(
-        "-cc",
-        "--custom-cookbook",
-        dest="custom_ami_cookbook",
-        help="Specifies the cookbook to use to build the AWS ParallelCluster AMI.",
-    )
-    pami.add_argument(
-        "--post-install",
-        dest="post_install_script",
-        help="Specifies the post install script to use to build the AWS ParallelCluster AMI.",
-    )
-    pami.add_argument(
-        "--no-public-ip",
-        dest="associate_public_ip",
-        action="store_false",
-        default=True,
-        help="Do not associate public IP to the Packer instance. Defaults to associate public ip",
+        "--image-name",
+        dest="image_name",
+        required=True,
+        help="Specifies the image stack name to use for building the AWS ParallelCluster AMI.",
```

Comment:
image stack name?  do you mean the stack name? or is it the image name?

Reply:
it is stack name, rename it to --name